content/unexpected-reboots.md

```diff
@@ -0,0 +1,441 @@
+
+1. Upgrade the Current OS:
+
+- Instructions for [Upgrading PopOS](/articles/upgrade-pop)
+
+- Instructions for [Upgrading Ubuntu](articles/upgrade-ubuntu)
+
+2. Boot from a Live Disk for Clean Install:
+
+- Make a [Live Disk](/articles/live-disk)
+
+3. Access the [BIOS/Boot Menu](/articles/boot-menu)
+
+4. Reinstall the OS:
+
+- [Install Pop](/articles/install-pop)
+
+- [Install Ubuntu](/articles/install-ubuntu) 
+
+### RAM Issues
+
+The first step in the boot process is to assess the available hardware and load required resources into memory.
+
+If your system is not POSTing (Power On Self Test), typically you'll get symptoms like the following:
+
+1. The system powers on, but nothing loads.
+
+2. The system gets to a splash screen/logo, but crashes shortly afterwards. 
+
+It may reach a login screen or desktop, but the system is unstable.
+
+This usually indicates an issue with RAM (Random Access Memory). Despite the name, the system tends to use similar areas of RAM when booting (for example, the RAM module in Slot 1 is accessed first). The earlier in the boot process the system hits bad sectors of RAM, the sooner the system will freeze or reboot.
+
+If the system is booting to a desktop, the first troubleshooting step is to run stress tests on the memory. If the stress test completes, it will show error messages for your RAM. If the test doesn't complete it will likely trigger the issue, and indicate that RAM is to blame.
+
+#### To Test RAM
+
+The application Memtester can be used to scan the RAM sticks for bad sectors. Memtester can be run from your booted OS, or from a Live Environment. This can be a [Live USB](/articles/live-disk), or the [Recovery Partition](/articles/pop-recovery)
+
+1. Install Memtester
+
+Open a `Terminal` with <kbd>SUPER</kbd>+<kbd>T</kbd> (Pop!\_OS) or <kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>T</kbd> (Ubuntu) run the following commands:
+
+```bash
+sudo apt update
+sudo apt install memtester
+```
+
+The first command will make sure your package lists are up to date.
+The second will install the application memtester.
+
+2. Once memtester is installed, instruct it to test **most** of the RAM, leaving enough for the OS to continue running.
+
+For example, if your system has 16GB of RAM, test 12GB or 14GB.
+
+To start the test, run these `Terminal` commands:
+
+```bash
+sudo memtester 12GB 3 > ~/memtester.txt
+```
+
+This command will run memtester and test 12GB of RAM 3 times, then save the output to a file called "memtester.txt" in your Home folder.
+
+> **IMPORTANT NOTE:** Close all applications other than Terminal when you run memtester. This will likely take a few hours (the application runs multiple passes over the memory) so it would be best to run it overnight.
+
+
+### Thermals
+
+Modern hardware is designed to shut systems down when they reach temperatures that may be damaging to the internal components. Typically these thresholds are in the upper 80s or 90s Celsius, depending on hardware.
```

Comment:
Either "Typically" should have a comma after it, or it should be moved after the word "are."

Reply:
I'll fix that.